### PR TITLE
[PRODX-24170]: added tests for AddCloudInstance component

### PR DESCRIPTION
**Pull request by AntonNos**
Screenshot 2022-08-15 at 00 30 54

### CODE REVIEW

**Comment by stefcameron** [comment]
👀

**Review by stefcameron — CHANGES_REQUESTED** [review]
I'll post an update to this branch soon.

**Comment by stefcameron** [comment]
With my changes, we're at

image

Just one statement is not covered:

image

But we don't need to worry about that one.

File: src/renderer/components/GlobalPage/AddCloudInstance.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ export const AddCloudInstance = ({ onAdd, onCancel }) => {
     let newCloud = new Cloud(normUrl);
     newCloud.name = clusterName;
     const statusListener = () => {
+      // Looks like we don't get here
```
[the remark this answers is not in the document]
Comment (AntonNos) [reply src/renderer/components/GlobalPage/AddCloudInstance.js]: Got it.

**Comment by AntonNos** [comment]
> With my changes, we're at
> 
> image
> 
> Just one statement is not covered:
> 
> image
> 
> But we don't need to worry about that one.

Great! Thanks for helping me with that.

**Review by stefcameron — APPROVED** [review]

**Pull request merged**